### PR TITLE
Feat/talks and speakers

### PR DESCRIPTION
**Pull request by anyulled**

### CODE REVIEW

**Comment by anyulled** [comment]
### Speakers
image
image
image
image
### Talks
image

File: 2020/_data/speakers.json [hunk 1]
```diff
@@ -1,61 +1,131 @@
...
-      "biography": "Dr. Venkat Subramaniam is an award-winning author, founder of Agile Developer, Inc., creator of agilelearner.com, and an instructional professor at the University of Houston. He has trained and mentored thousands of software developers in the US, Canada, Europe, and Asia, and is a regularly-invited speaker at several international conferences. Venkat helps his clients effectively apply and succeed with sustainable agile practices on their software projects. Venkat is a (co)author of multiple technical books, including the 2007 Jolt Productivity award winning book Practices of an Agile Developer. You can find a list of his books at agiledeveloper.com. You can reach him by email at venkats@agiledeveloper.com or on twitter at @venkat_s.",
       "enabled": "1",
-      "homepage": "http://agiledeveloper.com",
-      "image": "assets/img/speakers/venkat-subramaniam.jpg",
-      "ref": "VmVua2F0U3VicmFtYW5pYW12ZW5rYXRzQGFnaWxlZGV2ZWxvcGVyLmNvbQ",
-      "url": "infoSpeaker.html?ref=VmVua2F0U3VicmFtYW5pYW12ZW5rYXRzQGFnaWxlZGV2ZWxvcGVyLmNvbQ",
-      "twitter": "@venkat_s"
+      "name": "Ixchel Ruiz",
+      "company": "Karakun AG",
+      "description": "",
```
Comment (wedamartinez) [inline 2020/_data/speakers.json]: We don't have descriptions anymore?

Comment (anyulled) [reply 2020/_data/speakers.json]: image
added them now

Comment (abelsalgadoromeroWD) [reply 2020/_data/speakers.json]: la photo de Vaughn es la de este año?

Comment (anyulled) [reply 2020/_data/speakers.json]: ahora si
image
estos tres también los actualicé
image

**Comment by cougil** [comment]
@anyulled please do not remove Venkat, Trisha and Andrés they are confirmed and they are relevant enough and they're already published and announced. Also I think the description of Alex I think is wrong and we are missing the description of Grace

**Comment by anyulled** [comment]
image

**Pull request merged**